unpackaged/config/trial/profiles/Admin.profile

```diff
@@ -74,9 +66,25 @@
     <layoutAssignments>
         <layout>hed__Behavior_Involvement__c-%%%NAMESPACE%%%K12 Kit Behavior Involvement Layout</layout>
     </layoutAssignments>
+    <layoutAssignments>
+        <layout>hed__Behavior_Involvement__c-%%%NAMESPACE%%%K12 Kit Behavior Involvement Layout</layout>
+        <recordType>hed__Behavior_Involvement__c.Incident_Involvement</recordType>
```

Comment:
@jstvz do we need a %%%NAMESPACED_ORG%%% for record type here. I'm assuming not since its and EDA object.